tox.ini

```diff
@@ -0,0 +1,16 @@
+[tox]
+envlist = py27,py34
+
+[testenv]
+commands = python setup.py test
+setenv =
+       XDG_CACHE_HOME={envtmpdir}/
+# NOTE:
+# * as of 2016-09-20 pyac 1.4 does not support python3, so we must use
+#   the github repository
+# * as of 2016-09-20 python-megaplan does not support python3, so a
+#   github repository with the needed fixes has been created
+deps = pysimplesoap
+     py34: git+https://github.com/kostajh/pyac.git
+     py34: git+https://github.com/wfranzini/python-megaplan.git
```

Comment:
pyac 1.5 was just released. Any reason you haven't made a PR to upstream python-megaplan?